twistlock/requirements.in

```diff
@@ -0,0 +1 @@
+python-dateutil==2.8.0
```

Comment:
If it's on the base agent I don't think it needs to be here too. On the other hand, if it's only used here It does not need to be an agent requirement

Reply:
I tried that but some tests fail. 

- without this here `ddev test twistlock` fails with `ImportError: No module named dateutil`
- without  it in base requirement, `ddev test twistlock` succeeds but `ddev validate dep` fails with `Dependency `python-dateutil` for check `twistlock` missing from the embedded environment`

Reply:
Let me know if I'm missing something here and what would I need to change if any.